### PR TITLE
Include main line train travel EFs (France)

### PR DESCRIPTION
**Pull request by greenercode**
**Is this a new addition or an update to an existing factor?**

- [x] New

**What category or categories does it affect?**

rail travel 

**What?**

Support main line train travel (region: France)

**Why?**

Up until now, the OEFDB did not support these main french rail travel emission factors.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1011,13 +1011,21 @@ Transport,Rail Travel,passenger_train-route_type_na-fuel_source_electricity,Elec
...
 Transport,Rail Travel,passenger_train-route_type_light_rail_and_tram-fuel_source_na,Light rail and tram passenger train,kg-CO2e/pkm,0.02781,,BEIS,2021,GB,2021/07/23,Emission intensity for light rail and tram passenger train including fuel consumption. The light rail factors were based on an average of factors for a range of UK tram and light rail systems. Published by BEIS in June 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
+Transport,Rail Travel,passenger_train-route_type_mainline-route_location_na-fuel_source_na,Mainline train,kg-CO2e/pkm,0.00529,20,ADEME,2020,FR,2021/11/30,"Emission intensity for mainline train train including upstream electricity generation and/or fuel combustion. This emission factor does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks, stations, etc.). Provided by the GT Transport Base Carbone & InfoGES SNCF. Published by the Agence de l'Environnement et de la Maîtrise de l'Énergie in May 2020.",https://www.bilans-ges.ademe.fr/en/accueil,upstream-fuel_combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1019]: There is double “train”s in the description.

Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:1019]: It seems that we do not have the “route_location” for other ids in this category.

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1011,13 +1011,21 @@ Transport,Rail Travel,passenger_train-route_type_na-fuel_source_electricity,Elec
...
 Transport,Rail Travel,passenger_train-route_type_long_distance-fuel_source_electricity,Electric remote passenger train,kg-CO2e/pkm,0.00948,,UBA,2020,DE,2021/09/15,Emission intensity of long-distance electric passenger train including upstream. Data from the Öko-Institut.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={A4FFA0CD-2550-4435-BBDE-00C6F1A0B22F},upstream-electricity_consumption
+Transport,Rail Travel,passenger_train-route_type_high_speed-route_location_na-fuel_source_na,High speed train ,kg-CO2e/pkm,0.00173,20,ADEME,2020,FR,2021/11/30,"Emission intensity for high speed train including upstream electricity generation and/or fuel combustion. 2019. This emission factor does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks, stations, etc.). Provided by the GT Transport Base Carbone & InfoGES SNCF. Published by the Agence de l'Environnement et de la Maîtrise de l'Énergie in May 2020.",https://www.bilans-ges.ademe.fr/en/accueil,upstream-fuel_combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1014]: Do we need 2019 in the description?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1011,13 +1011,21 @@ Transport,Rail Travel,passenger_train-route_type_na-fuel_source_electricity,Elec
...
+Transport,Rail Travel,passenger_train-route_type_rer_and_transilien-route_location_paris_and_suburbs-fuel_source_na,RER (Paris area) and Transilien (suburban-train lines),kg-CO2e/pkm,0.0041,20,ADEME,2020,FR PAR,2021/11/30,"Emission intensity for RER (Paris area) and Transilien (suburban-train lines) including upstream electricity generation and/or fuel combustion. 2019. This emission factor does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks, stations, etc.). Provided by the GT Transport Base Carbone &  RATP - SNCF. Published by the Agence de l'Environnement et de la Maîtrise de l'Énergie in May 2020.",https://www.bilans-ges.ademe.fr/en/accueil,upstream-fuel_combustion
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1024]: Does the source mention “upstream electricity generation”? I wonder this refers to fuel upstream for elec generation or fuel combustion for electricity generation.

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (greenercode) [reply OpenEmissionFactorsDB.csv:1024]: It is unclear - this is what we decided with @patstirling yesterday.

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv]: There is an extra space at the end of the name field and also commas int he description.

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -1011,13 +1011,21 @@ Transport,Rail Travel,passenger_train-route_type_na-fuel_source_electricity,Elec
...
 Transport,Rail Travel,passenger_train-route_type_long_distance-fuel_source_electricity,Electric remote passenger train,kg-CO2e/pkm,0.00948,,UBA,2020,DE,2021/09/15,Emission intensity of long-distance electric passenger train including upstream. Data from the Öko-Institut.,https://www.probas.umweltbundesamt.de/php/prozessdetails.php?id={A4FFA0CD-2550-4435-BBDE-00C6F1A0B22F},upstream-electricity_consumption
+Transport,Rail Travel,passenger_train-route_type_high_speed-fuel_source_na,High speed train,kg-CO2e/pkm,0.00173,20,ADEME,2020,FR,2021/11/30,"Emission intensity for high speed train including upstream electricity generation and/or fuel combustion.This emission factor does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks, stations, etc.). Provided by the GT Transport Base Carbone & InfoGES SNCF. Published by the Agence de l'Environnement et de la Maîtrise de l'Énergie in May 2020.",https://www.bilans-ges.ademe.fr/en/accueil,upstream-fuel_combustion
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv:1014]: There are commas in the description field

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Good job Stephane! Thank you!

Please fish those commas out of description and name fields :)

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: Hi both - here's the mothodology that ADEME cites from SNCF: http://medias.sncf.com/sncfcom/pdf/co2/Information_CO2_des_prestations_de_transport_Methodologie_generale.pdf - this is what ADEME refer to in their methodology. The key bit is "(Consommation d’électricité x facteur d’émission de CO2e de l’électricité à usage transport + Consommation gasoil x Facteur d’émission du gasoil) / Voyageurs x km =
Emission d’un voyageur par type de train exprimé en gramme de CO2e /km". This implies that these EFs are the addition of the emissions of the electricity used (which is implicitly upstream) and the combustion of fuel in use-phase (but NOT upstream costs of fuel). How would you advise this be reflected in the lca field @maryarbab ?

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv]: These do feel like a very long descriptions. Remember that the description field only needs to provide detail not already in the other fields that a user might need to help choose the right EF, or clarify source (e.g. by exploding initialisms like ADEME). Suggest we remove the reference to 2019 and remove "This emission factor does not include the manufacture of rolling stock or emissions related to the infrastructure (railway tracks stations etc.). Provided by the GT Transport Base Carbone & InfoGES SNCF." from all EFs

Comment (greenercode) [reply OpenEmissionFactorsDB.csv]: Addressed this.

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv]: My suggestion is: electricity_generation-fuel_combustion.

**Review by patstirling — APPROVED** [review]

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv]: Do we agree to have electricity_generation-fuel_combustion as the LCA activity? It still seems to be upstream-fuel_combustion in the lca activity field.

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by maryarbab — APPROVED** [review]

**Pull request merged**